nss-day-cohort-47/reactive-nutshell-palm-beach-pilots
# Cc event friends

# Description
Added Event sorting so that dates that have passed are not shown
Added event sorting so latest date shows first
Added filtering so that current user's friends events also show in event list
Added conditional to check if event is current user or friend and if friend then italicize the event and remove edit and delete event buttons
Added cancel buttons to add event and edit event form


## Type of change


# Testing Instructions
User should be able to click on events and see them from soonest to latest. Past events should not show. Friends events are italicized and have edit and delete buttons removed. User should be able to cancel out of event add form and edit form.
# Checklist:
